src/main/java/training/commands/Elbow/Constants.java

```diff
@@ -0,0 +1,5 @@
+package training.commands.Elbow;
+
+public class Constants {
```

Comment:
```suggestion
public class ElbowConstants {
```
put this const in ElbowConstants in elbow package, don't seperate for commands and subsystem
Also don't commit from here